CHANGELOG.md

```diff
@@ -2,13 +2,16 @@
 
 ## XX.XX.XX - 2023-XX-XX
 
-## 20.30.0 - 2023-09-11
```

Comment:
Looks like you killed a release :O

Reply:
Ohh, me dumb. sorry

Reply:
But looks like the rebase wasn't done, since there was also a 20.30.1

Reply:
I _had_ killed a release lol, I don't know why the first rebase I did didn't bring that change, I had to rebase again.